### PR TITLE
MRF-51791 adapt sulvo to work with inventory.json

### PR DESCRIPTION
**Pull request by alexriusmrf**

### CODE REVIEW

File: json-lint/schemas/inventory/adservers/sulvo.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "$id": "#sulvo",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Sulvo AdServer",
+    "description": "Sulvo for touch",
+    "additionalProperties": true,
```
Comment (ettoreciprianmrf) [inline json-lint/schemas/inventory/adservers/sulvo.json]: This should be false

Comment (ettoreciprianmrf) [inline json-lint/schemas/inventory/adservers/sulvo.json]: this should be false

File: json-lint/schemas/inventory/adservers/sulvo.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                "scriptSrc": "$1",
+                "id": "$2"
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "const": "sulvo"
+        },
+        "scriptSrc": {
```
Comment (ettoreciprianmrf) [inline json-lint/schemas/inventory/adservers/sulvo.json]: Let's put this in the JSON pocket anyway

**Comment by florevallatmrf** [comment]
@alexriusmrf Hello, can you do this commit again over at https://github.com/Marfeel/MarfeelSchemas ?  🙏 
I'm about to make the files from MarfeelSchemas the ones we rely one, and delete the ones here. 
Thank you!

**Comment by alexriusmrf** [comment]
> @alexriusmrf Hello, can you do this commit again over at https://github.com/Marfeel/MarfeelSchemas ? 🙏
> I'm about to make the files from MarfeelSchemas the ones we rely one, and delete the ones here.
> Thank you!

https://github.com/Marfeel/MarfeelSchemas/pull/2

**Pull request merged**